shell/browser/ui/accelerator_util.cc

```diff
@@ -21,7 +21,9 @@ namespace accelerator_util {
 bool StringToAccelerator(const std::string& shortcut,
                          ui::Accelerator* accelerator) {
   if (!base::IsStringASCII(shortcut)) {
-    LOG(ERROR) << "The accelerator string can only contain ASCII characters";
+    LOG(ERROR) << "The accelerator string can only contain ASCII characters, "
+                  "invalid string: "
+               << shortcut;
```

Comment:
This looks good but I think it can be slightly improved:

While debugging I usually try to add quotes to the printed string so that I can tell if a string contains surrounding spaces. I also print the length of the string to understand if it contains non-printing characters.

Reply:
Done